octoprint_mrbeam/__init__.py

```diff
@@ -301,8 +296,23 @@ def initialize(self):
         self._do_initial_log()
         self._printer.register_user_notification_system(self.user_notification_system)
 
-        # Initialize the laser cutter mode service
-        self._laser_cutter_mode_service = laser_cutter_mode_service(self)
+    def update_laser_cutter_profile(self, profile):
+        self.laser_cutter_profile_service.save(profile, allow_overwrite=True, make_default=True)
+        self.laser_cutter_profile_service.select(profile['id'])
+
+    def get_laser_cutter_profile_for_current_configuration(self):
+        laser_cutter_mode = self.get_laser_cutter_mode()
+        device_series = self._device_info.get_series()
+        profile = laser_cutter_profiles.default_profile
+        if laser_cutter_mode == LaserCutterModeEnum.DEFAULT.value:
+            if device_series == "2C":
```

Comment:
The list of device series was removed from the plugin implementation because it leads to unneeded overhead every time we add a new series. Now the series comes from the /etc/mrbeam file always.

Reply:
I will add the Enum for the relevant device series though! Great point